clients/src/main/java/org/apache/kafka/common/record/Compressor.java

```diff
@@ -201,16 +201,24 @@ public void put(final byte[] bytes, final int offset, final int len) {
         }
     }
 
-    public void putRecord(long timestamp, byte[] key, byte[] value, CompressionType type, int valueOffset, int valueSize) {
+    /**
+     * @return CRC of the record
```

Comment:
So my thinking was to use CRC in the internal classes, since they already use CRC. I am using word 'checksum' in RecordMetadata and ConsumerRecord classes. I thought it is better to use a more general word, but I don't have strong opinion about this. @junrao what do you think: use of 'crc' or 'checksum' for field names in  RecordMetadata and ConsumerRecord?